### DISCUSSION TITLE
Sorting & grouping servers.

### DISCUSSION THREAD

**Discussion by codebycam** [pelican-dev/panel#26] [Ideas]
G'day, 

Just quickly want to say as a Pterodactyl user for quite some time, I'm very excited about Pelican to see more frequent development! 
I'd like to request something that was long sought-after in Pterodactyl, which is easily sorting & grouping servers on the main page. 

Sorting of servers could be as simple as drag and drop and this alone would tremendously improve the user experience when one has many servers. It would also be great to have grouping too, for example you could have two groups: "Production" & "Development" which would house different servers inside yet again improving the user experience. Grouping servers could be similar to sorting servers, but you drag and drop a server onto another to create the group (think creating groups for apps on a smartphone). This is just a huge quality of life feature that I feel has been missing from Pterodactyl from the get-go.

Here is a quick video by @smashyalts who created something similar in Pterodactyl as an open source addon.

https://github.com/pelican-dev/panel/assets/41496198/82608c4f-e501-477c-a6fb-76208d7dde0a

Cheers for taking the time to read this, and I hope it will be taken under consideration!

**Comment by smashyalts** [comment]
<3 thanks for the mention

**Comment by Stealthx3** [comment]
Some better methods for navigating large server and user lists in general would be nice, along with the ability to execute bulk actions on multiple servers (transfer to another node, suspend, bulk delete etc). Maybe a tag-based filtering search with auto-complete.

**Comment by lancepioch** [comment]
Thanks, please follow #249
